README.md

```diff
@@ -42,7 +42,9 @@ libc = "0.2"
 
 ## Rust version support
 
-The minimum supported Rust toolchain version is **Rust 1.13.0** . APIs requiring
+The minimum supported Rust toolchain version is currently **Rust 1.13.0**.
+(libc does not currently have any policy regarding changes to the minimum
+supported Rust version; such policy is a work in progress.) APIs requiring
```

Comment:
Would it be valuable to link to the specific issue where this policy is being discussed? Or maybe even a short description of "in the libc issues" or something, so people know where to go (if they don't come to this PR and find the backlink here, that is).

Reply:
@cole-h I don't think a link from the README to a github issue would be appropriate.

I would be happy to add some indication of where discussion is taking place, except that even that seems to be a matter of some consternation in places, and I'm concerned about such documentation being interpreted as normative (this is where the discussion *should* take place) rather than informative (this is where the discussion is in practice taking place).

Reply:
For now, I've pinned https://github.com/rust-lang/libc/issues/2850 so that repo visitors can notice there's a discussion on the libs-team issue tracker.

Reply:
OK, makes sense! Thank you both!

Reply:
@JohnTitor Good idea!